packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,15 +1,65 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, hasFocus, hasInput, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary,
+        paddingTop: 0,
+        paddingBottom: 0,
+        selectors: {
+          ':hover ': {
+            borderColor: palette.neutralPrimary
+          },
+          ':active': {
+            borderColor: palette.themePrimary
+          }
+        }
+      },
+      !hasFocus &&
+        hasInput && {
```

Comment:
I'm confused by this logic—what's happening here?

Reply:
Hi @Jahnp ! Thanks for commenting on my PR. So this logic specifically targets the scenario of an SearchBox having text but isn't currently focused on. Fabric 6 handles this by changing the border color AND allowing the magnifying glass to transition back into view. So per the toolkit interactive state called "Populated" in the Suite Header toolkit (scroll all the way to the bottom to "Secondary searchbox"), I'm removing the ability for the mag glass to transition back into view upon lost focus on a populated SearchBox.

Reply:
Oh my gosh, my bad—in my haste I read `hasInput` as `hasFocus`, so when I commented I thought I was seeing this:
```typescript
!hasFocus &&
  hasFocus && {
  ...
```

Your explanation makes 100% sense and you can completely disregard this comment.